### ISSUE TITLE
Add a SAFER Handover model

### ISSUE THREAD

**Issue by timcknowles** [timcknowles/epidur.io#10]

**Comment by freshehr** [comment]
@timcknowles -do you have documentation for the SAFE or SAFER handover tool? I had a look and could find some academic papers but nothing to define the tools themselves.

**Comment by timcknowles** [comment]
@freshehr yes I'll email you copy of the local implementation we use. This is done with paper, pen and a lever arch  folder at the moment. The form is based on the SAFER model you have found in the academic papers. I was wondering if openEHR had any boilerplate archetypes for handover already?
